### ISSUE TITLE
[crash] SIGSEGV in Body::GetPositionRelTo

### ISSUE THREAD

**Issue by hexagonrecursion** [pioneerspacesim/pioneer#5711]
### Observed behaviour

The game crashes soon after I jump into this specific system

Stack trace:
```c++
#0  0x000055555579b92c in Body::GetPositionRelTo (this=0x0, relToId=...) at /run/build/pioneer/src/Body.cpp:203
#1  0x000055555579bb97 in Body::GetPositionRelTo (this=<optimized out>, relTo=<optimized out>) at /run/build/pioneer/src/Body.cpp:221
#2  0x000055555568b1d4 in AICmdFlyAround::TimeStepUpdate (this=0x55555dc29350) at /run/build/pioneer/src/ShipAICmd.cpp:1519
#3  0x0000555555813469 in Ship::AITimeStep (this=0x55555dc26380, timeStep=timeStep@entry=0.0166666675) at /run/build/pioneer/src/Ship-AI.cpp:37
#4  0x000055555577d8a8 in ShipController::StaticUpdate (this=0x55555a22f600, timeStep=0.0166666675) at /run/build/pioneer/src/ship/ShipController.cpp:11
#5  0x000055555567a56a in Ship::StaticUpdate (this=0x55555dc26380, timeStep=0.0166666675) at /run/build/pioneer/src/Ship.cpp:1215
#6  0x000055555569caa5 in Space::TimeStep (this=0x55555cf630a0, step=step@entry=0.0166666675) at /run/build/pioneer/src/Space.cpp:1042
#7  0x00005555557c5a7f in Game::TimeStep (this=0x55555a2160d0, step=0.0166666675) at /usr/include/c++/12.2.0/bits/unique_ptr.h:191
#8  0x00005555556466c2 in GameLoop::Update (this=0x555555bd63e0, deltaTime=0.0165723618) at /run/build/pioneer/src/Pi.cpp:944
#9  0x00005555555f5eb4 in Application::Run (this=0x555555bd5ff0) at /run/build/pioneer/src/core/Application.cpp:192
#10 0x00005555555d528c in main (argc=1, argv=<optimized out>) at /run/build/pioneer/src/main.cpp:186
```

### Expected behaviour

No crash

### Steps to reproduce

1. Load the attached save
2. Unpause
3. Wait 30 seconds
4. Crash

__My pioneer version (and OS):__ 20231204 (and Fedora release 39)

__My output.txt (required) and game save (optional, but recommended)__

[output.txt](https://github.com/pioneerspacesim/pioneer/files/13917299/output.txt)
[Alice 222.zip](https://github.com/pioneerspacesim/pioneer/files/13917283/Alice.222.zip)

**Comment by Gliese852** [comment]
Please specify the commit hash you built from.

**Comment by hexagonrecursion** [comment]
> Please specify the commit hash you built from.

I am playing a faltpak built from [this commit](https://github.com/flathub/net.pioneerspacesim.Pioneer/commit/c7ad7eea44402d847834ea6776fcc5091f0d797a)

This uses pioneer release [20230203](https://github.com/pioneerspacesim/pioneer/tree/20230203) with a patch to prevent crashing when saving the game (see https://github.com/flathub/net.pioneerspacesim.Pioneer/pull/7 and https://github.com/pioneerspacesim/pioneer/issues/5595)

**Comment by Gliese852** [comment]
Then it's very similar to what was fixed in #5564

**Comment by Gliese852** [comment]
Yes, in your save, 3 NPC ships have an active autopilot command to fly around the body, with an invalid index. It was definitely already fixed, in the PR above.

I cut out these autopilots from your save, and now it doesn’t crash. But when jumping to another system, this can happen again.
[alice-fixed.zip](https://github.com/pioneerspacesim/pioneer/files/13924009/alice-fixed.zip)
